<launch>
  <arg name="kinect_type" default="Kinect_v2"/>
  <arg name="kinect_name" default="kinect2"/>
  <arg name="camera_frame" default="/$(arg kinect_name)_link"/>
  <arg name="output_file" default=""/>
  <arg name="serial" default="501427243142"/>

  <node name="kinect_points_calibration" pkg="kinect_extrinsics_simple" type="simple_singlepoints_calib.py" output="screen">
    <param name="camera_name" value="$(arg kinect_name)" />
    <param name="camera_frame" value="$(arg camera_frame)" />
    <param name="output_file" value="$(arg output_file)" />
    <param name="serial" value="$(arg serial)" type="str" /> 
    <param name="kinect_type" value="$(arg kinect_type)" /> 
  </node>
</launch>
